### PR TITLE
Adding a TeamCity Continuous Delivery post

### PR DESCRIPTION
**Pull request by stack72**

### CODE REVIEW

File: content/docs/guides/continuous-delivery/teamcity.md [hunk 1]
```diff
@@ -0,0 +1,156 @@
...
+    userguides:
+        parent: cont_delivery
+        weight: 1
+
+aliases:
+- /docs/reference/cd-jetbrains-teamcity/
+- /docs/console/continuous-delivery/jetbrains-teamcity/
+---
+
+This page details how to use [JetBrains TeamCity](https://www.jetbrains.com/teamcity/) to deploy some sample infrastructure
```
Comment (spara) [inline content/docs/guides/continuous-delivery/teamcity.md]: "deploy a sample infrastructure"

File: content/docs/guides/continuous-delivery/teamcity.md [hunk 2]
```diff
@@ -0,0 +1,156 @@
...
+be to use the [Kotlin DSL](https://www.jetbrains.com/help/teamcity/kotlin-dsl.html). When creating a new project via the UI,
+we will see the following creation wizard:
+
+![TeamCity Project Creation Wizard](/images/docs/reference/teamcity/new-project.png)
+
+We are going to follow the TeamCity project setup `From a repository URL`. This means we can enter the URL to the 
+[Pulumi Examples Repository](https://github.com/pulumi/examples). This specific respository is open source, so we do not
+need to enter a `Username` or a `Password`. We can then `Proceed`.
+
+TeamCity will tell us if it can successfully connect to the repository. If it can, then it will ask us to give this project
```
Comment (spara) [inline content/docs/guides/continuous-delivery/teamcity.md]: delete "give", replace with  "name"

File: content/docs/guides/continuous-delivery/teamcity.md [hunk 3]
```diff
@@ -0,0 +1,156 @@
...
+a name and to create a [build configuration])(https://www.jetbrains.com/help/teamcity/build-configuration.html) name.
```
Comment (spara) [inline content/docs/guides/continuous-delivery/teamcity.md]: delete "a name"

File: content/docs/guides/continuous-delivery/teamcity.md [hunk 4]
```diff
@@ -0,0 +1,156 @@
...
+
+![TeamCity Project Naming](/images/docs/reference/teamcity/project-name.png)
+
+Let's call the project `Pulumi Example` and let's create a build configuration called `Development Environment`. We can
```
Comment (spara) [inline content/docs/guides/continuous-delivery/teamcity.md]: delete second "let's" in the sentence

File: content/docs/guides/continuous-delivery/teamcity.md [hunk 5]
```diff
@@ -0,0 +1,156 @@
...
+
+### Installing Pulumi
+
+Again, we are going to chose a `Command Line` build runner and we can create the build step as follows
+
+![Pulumi Install](/images/docs/reference/teamcity/pulumi-install.png)
+
+### Restoring NPM Dependencies
+
+The next build step we are going to create is to restore the NPM dependencies required to deploy our application. Let's create
```
Comment (spara) [inline content/docs/guides/continuous-delivery/teamcity.md]: delete "we are going to create"

**Comment by joeduffy** [comment]
@stack72, this is awesome!

Quick question: Is there any reason the introductory sentence explicitly calls out AWS? Upon my first read, this gave me the impression that Pulumi only works with TeamCity when using with AWS.

**Comment by stack72** [comment]
@joeduffy nope - more than happy to change it to say it works with all clouds - I didn’t read it that way so it’s great feedback

Will update it

File: content/docs/guides/continuous-delivery/teamcity.md [hunk 6]
```diff
@@ -0,0 +1,186 @@
...
+meta_desc: "This page provides an overview of how to use Pulumi with JetBrains TeamCity."
+
+menu:
+    userguides:
+        parent: cont_delivery
+        weight: 1
+
+aliases:
+- /docs/reference/cd-jetbrains-teamcity/
+- /docs/console/continuous-delivery/jetbrains-teamcity/
```
Comment (justinvp) [inline content/docs/guides/continuous-delivery/teamcity.md]: @stack72, if this is a brand new page, these aliases should be deleted. The aliases are only necessary for existing pages that have changed paths within the site.

**Pull request merged**